tests/lib/rules/no-ember-testing-in-module-scope.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const rule = require('../../../lib/rules/no-ember-testing-in-module-scope');
+const RuleTester = require('eslint').RuleTester;
+
+const messages = rule.meta.messages;
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+});
+
+ruleTester.run('no-ember-testing-in-module-scope', rule, {
+  valid: [
+    {
+      code: 'foo.testing = true;'
+    },
+    {
+      code: `
+        import Ember from 'ember';
+
+        export default Ember.Component.extend({
+          init() {
+            this.isTesting = Ember.testing;
```

Comment:
IMHO we should also forbid assignment expressions where the right hand side is `Ember.testing`...

Reply:
d'oh